### PR TITLE
mpv: update to 0.35.1.

### PR DESCRIPTION
**Pull request by daniel-eys**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/mpv/template [hunk 1]
```diff
@@ -41,7 +41,7 @@ license="GPL-2.0-or-later"
 homepage="https://mpv.io"
 changelog="https://github.com/mpv-player/mpv/releases"
 distfiles="https://github.com/mpv-player/${pkgname}/archive/v${version}.tar.gz"
```
Comment (mhmdanas) [inline srcpkgs/mpv/template:43]:
```suggestion
distfiles="https://github.com/mpv-player/mpv/archive/v${version}.tar.gz"
```

Comment (daniel-eys) [reply srcpkgs/mpv/template]: done, thanks

**Comment by daniel-eys** [comment]
it is enabled or am I missing something?
https://github.com/void-linux/void-packages/actions/runs/4046324848/jobs/6958967610#step:7:277

**Comment by mhmdanas** [comment]
Shouldn't https://github.com/void-linux/void-packages/pull/41954/files#diff-8b6fb7d0e7b23a3c60f6d80062073fbf5d9d3c48dc2255e9020951ad8ea42943R65 be changed to:
```sh
vmove "usr/lib/*.so"
```
?

**Comment by daniel-eys** [comment]
> Shouldn't line 65 in the template be changed to:
> 
> ```shell
> vmove "usr/lib/*.so"
> ```
> 
> ?

indeed, quoting is perferred here eventhough it happens to work without. PR updated

**Pull request merged**